backend/src/API/mailAPI.ts

```diff
@@ -194,3 +195,26 @@ export const sendTECReminder = async (
   } pending this semester.\n${reminder}\nTo submit your TEC, please visit https://idol.cornelldti.org/forms/teamEventCredits.`;
   return emailMember(req, member, subject, text);
 };
+
+/**
+ * Send an email reminder to members who do not have a coffee chat blackout
+ * @param req - The request made when sending the email
+ * @param member - The member being sent the email
+ * @returns - The response body containing information of the member being sent the email
+ */
+export const sendCoffeeChatReminder = async (
+  req: Request,
+  member: IdolMember
+): Promise<AxiosResponse> => {
+  const subject = 'Coffee Chat Reminder';
+
+  const isNewbie = await checkMemberMeetsCategory(member.email, member.email, 'a newbie').then(
```

Comment:
Oh actually, we can use `member.semesterJoined` now to figure this out! I think this would involve adding some util function to figure out the current semester and compare it with the field.